tests/test_gossip.py

```diff
@@ -567,110 +570,167 @@ def test_gossip_query_channel_range(node_factory, bitcoind):
 
     assert block23 == block12 + 1
 
-    # l1 asks for all channels, gets both.
-    ret = l1.rpc.dev_query_channel_range(id=l2.info['id'],
-                                         first=0,
-                                         num=1000000)
-
-    assert ret['final_first_block'] == 0
-    assert ret['final_num_blocks'] == 1000000
-    assert ret['final_complete']
-    assert len(ret['short_channel_ids']) == 2
-    assert ret['short_channel_ids'][0] == scid12
-    assert ret['short_channel_ids'][1] == scid23
+    # Asks l2 for all channels, gets both.
+    msgs = l2.query_gossip('query_channel_range',
+                           '06226e46111a0b59caaf126043eb5bbf28c34f3a5e332a1fc7b2b73cf188910f',
+                           0, 1000000)
+    encoded = subprocess.run(['devtools/mkencoded', '--scids', '00', scid12, scid23],
+                             check=True,
+                             timeout=TIMEOUT,
+                             stdout=subprocess.PIPE).stdout.strip().decode()
+    # reply_channel_range == 264
```

Comment:
265 ? https://github.com/lightningnetwork/lightning-rfc/blob/master/07-routing-gossip.md#the-query_channel_range-and-reply_channel_range-messages